src/mockedData/mockedData.ts

```diff
@@ -203,3 +207,27 @@ export const tagsArr: Array<string> = [
   "Fresh food",
   "Lemons",
 ];
+
+export const categoriesArr: Array<{ value: string; checked: boolean }> = [
+  { value: "Fish", checked: false },
+  { value: "Vegetable", checked: false },
+  { value: "Fruit", checked: false },
+  { value: "Pasta", checked: false },
+];
+
+export const brandArr: Array<{ value: string; checked: boolean }> = [
+  { value: "Grocery Tarm Fields", checked: false },
+  { value: "ArabVanya", checked: false },
+  { value: "Iuda Production", checked: false },
+  { value: "Lanselot Farm", checked: false },
+  { value: "Tor Farm", checked: false },
+];
```

Comment:
Why is it static?

Reply:
ah, it's like a moked folder for static values ​​only. I understand that these arrays need to be moved to a new folder Date?

Reply:
You need to take categories and brands from products. Why do you need mocked categories/brands at all?

Reply:
To display the sidebar in screen and to compare with the data and change the status of the clicked ones, I take most popular , so I do not need all the categories for example, there are too many of them.